src/socialMediaShare/constants.js/index.js

```diff
@@ -0,0 +1,4 @@
+export const SharedContent = {
+  URL: "https://covid-19-au.github.io/",
```

Comment:
Any particular reason we separate the last three files? I feel like we can merge the three of them?